src/mod_auth_cas.c

```diff
@@ -22,7 +22,9 @@
  *
  */
 
+#ifdev HAVE_ERROR_H
```

Comment:
Is "#ifdev" a mistake (should be "#ifdef")?  How did this successfully compile ?